client/blocks/jetpack-backup-creds-banner/index.jsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+import getRewindState from 'state/selectors/get-rewind-state';
+import QueryRewindState from 'components/data/query-rewind-state';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug, isJetpackSite } from 'state/sites/selectors';
+
+class JetpackBackupCredsBanner extends Component {
+	static propTypes = {
+		// Banner location, passed to tracks event, single word to follow tracks naming conventions.
+		location: PropTypes.string.isRequired,
+	};
+
+	render() {
+		const { isJetpack, rewindState, siteId, slug, translate } = this.props;
+		const location = this.props.location || 'nolocation';
+		const event = `calpyso_jetpack_backup_credential_banner_${ location }_click`;
```

Comment:
I've suggested a fix for a typo here, but I also think this event name is too long. Also, it's generally not recommended to dynamically build event names, as that makes them difficult to search for later. Do you think we could omit some words? Maybe `jetpack` is obsolete, and the `location` could be moved to an event prop?

```suggestion
		const event = `calypso_jetpack_backup_credential_banner_${ location }_click`;
```

Reply:
CALPYSO!

>  location could be moved to an event prop?

Not sure if <Banner/> allows that. If not I'll change it to pass the whole event name in, meaning the search will work.

Reply:
Yeah, `Banner` doesn't support extra event props out of the box. 

Note that the event name here is passed as an event prop too, so I'd assume there's no need for the `calypso` prefix either.